app/Services/MapboxGeocodingService.php

```diff
@@ -26,4 +29,35 @@ public function getCoordinatesFromApi(string $cityName, string $countryName): ar
             return [];
         }
     }
+
+    public function getPlaceFromApi(string $lat, string $long): array
+    {
+        $token = env("MAPBOX_TOKEN");
```

Comment:
use `config()` instead of `env()`.

`
if you execute the config:cache command during your deployment process, you should be sure that you are only calling the env function from within your configuration files. Once the configuration has been cached, the .env file will not be loaded and all calls to the env function will return null.
`
https://laravel.com/docs/10.x/helpers#method-env